examples/counter/sources/counter.move

```diff
@@ -13,7 +13,7 @@ module rooch_examples::counter {
         counter.value = counter.value + 1;
      }
 
-     public entry fun init(account: signer){
+     public entry fun init_entry(account: signer){
```

Comment:
Is this a temporary method, or will there never be a function like `entry fun init()` in the future?

Reply:
Yes, the init function name cannot be occupied

Reply:
Please help refactor the counter example according to the standard of the init method, or you can start a new PR to implement it.